apex/contrib/group_norm/group_norm.py

```diff
@@ -31,6 +33,15 @@ def one_time_warning(msg: str):
         one_time_warning.has_been_called = True
         print(f'\033[93m{msg}\033[0m')  # hightlight with yellow color
 
+
+@functools.cache
+def get_cc_and_sm_count(device_index: int):
+    props = torch.cuda.get_device_properties(device_index)
+    CC = (props.major, props.minor)
```

Comment:
QQ: what would CC mean?

Reply:
It is compute capability. Basically a sm version.